sdcm/utils/raft/__init__.py

```diff
@@ -141,6 +143,9 @@ def get_all_messages_timeouts(self, operation: TopologyOperations):
     def call_read_barrier(self):
         ...
 
+    def search_inconsistent_host_ids(self) -> list[str]:
+        ...
```

Comment:
You can get rid of the dots by providing a doc string

Reply:
the elipsis needs here, to avoid IDE warning that wrong return type., but doc string also added

Reply:
correct approcha would be using ABC and mark it as abstract, but this is fine for now too